deepinv/physics/mri.py

```diff
@@ -126,7 +126,7 @@ def check_mask(self, mask: Tensor = None) -> None:
         r"""
         Updates MRI mask and verifies mask shape to be B,C,H,W.
 
-        :param torch.nn.Parameter, float MRI subsampling mask.
+        :param torch.nn.Parameter, float mask: MRI subsampling mask.
```

Comment:
```suggestion
        :param torch.Tensor, float mask: MRI subsampling mask.
```

here I don't really understand why mask could be a float (for MRI)?

Reply:
in this file (and in other places) we use both 
```python
from torch import Tensor
```
and then somewhere later
```python
:param torch.Tensor x:
```
I guess we should keep one of both, which one should we choose?

Reply:
> here I don't really understand why mask could be a float (for MRI)?

You're right, this is already fixed in #287 anyway as these functions are reorganised.

> I guess we should keep one of both, which one should we choose?

We should standardise it, happy with either. For type checking deepinv types I've started doing

```python
from __future__ import annotations
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from deepinv.physics.forward import Physics #etc.
```

Reply:
> here I don't really understand why mask could be a float (for MRI)?

You're right, this is already fixed in #287 anyway as these functions are reorganised.

> I guess we should keep one of both, which one should we choose?

We should standardise it, happy with either. For type checking deepinv types I've started doing

```python
from __future__ import annotations
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from deepinv.physics.forward import Physics #etc.
```

Reply:
I'll keep just torch.nn.Tensor and remove Tensor for the moment. Since this is an issue that appears throughout the library, I think we should solve it on a dedicated PR, instead of bit by bit in each PR.